db/compaction/tiered_compaction_test.cc

```diff
@@ -445,8 +445,9 @@ TEST_P(TieredCompactionTest, RangeBasedTieredStorageUniversal) {
     }
     ASSERT_OK(Flush());
   }
-  ASSERT_OK(dbfull()->WaitForCompact(
-      true));  // make sure the compaction is able to finish
+  ASSERT_OK(
+      dbfull()
+          ->WaitForCompact());  // make sure the compaction is able to finish
```

Comment:
Nit: if you move the comment above the code, it will be two source lines instead of three